src/style.css

```diff
@@ -0,0 +1,22 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.firstBlock {
+  background-color: red;
+  height: 100px;
+  width: 100%;
```

Comment:
[DRY](https://www.google.com/search?q=DRY&oq=DRY&aqs=chrome..69i57j0l7.1684j0j7&sourceid=chrome&ie=UTF-8) - do not repeat code